packages/web/docs/src/components/masking-scrollview.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useLayoutEffect, useReducer, useRef, useState } from 'react';
+
+export type MaskingScrollviewFade = {
+  y?: boolean;
+  x?: boolean;
+};
+
+export interface MaskingScrollviewProps {
+  fade: MaskingScrollviewFade;
+  children?: React.ReactNode;
```

Comment:
we can just provide `null`

```suggestion
  children: React.ReactNode;
```